test/fixtures/lightscript/named-arrow-functions/skinny-async-gen/actual.js

```diff
@@ -0,0 +1,2 @@
+fn() -/*>
+  yield await x
```

Comment:
We should also probably test that this raises a SyntaxError:

```
f() =*/>
  <- yield g()
```

If it doesn't do that by default, let's make it a friendly one 😄